src/BrowserKitDriver.php

```diff
@@ -23,7 +23,15 @@
 use Symfony\Component\DomCrawler\Field\InputFormField;
 use Symfony\Component\DomCrawler\Field\TextareaFormField;
 use Symfony\Component\DomCrawler\Form;
-use Symfony\Component\HttpKernel\Client as HttpKernelClient;
+use Symfony\Component\HttpKernel\HttpKernelBrowser;
+
+if (!class_exists('Symfony\Component\BrowserKit\AbstractBrowser')) {
+    class_alias('Symfony\Component\BrowserKit\Client', 'Symfony\Component\BrowserKit\AbstractBrowser');
```

Comment:
Please don't create class aliases for Symfony classes. This could break other packages detecting versions of Symfony based on the existence of classes. We should not be declaring classes in the `Symfony` namespace.